bleachbit/DeepScan.py

```diff
@@ -26,6 +26,20 @@
 
 import os
 import re
+import sys
+import unicodedata
+
+
+def normalize_filename(fn):
+    """
+    macOS uses decomposed UTF-8 to store filename. This functions
+    recomposes them on macOS.
+    """
+    if 'darwin' == sys.platform:
```

Comment:
This gets evaluated for each pass through a busy loop, so to avoid a performance penalty, please redefine the function. For an example see `FileUtilities.expandvars()` in BleachBit version 1.12